### PR TITLE
api

### PR DESCRIPTION
**Pull request by ShiJinSi**

### CODE REVIEW

File: gj/2017-12-5/api.txt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+2.用户登录
+url:/user/login
+作用：进入用户登录页面
+
+3.用户注册
+url:/user/register
+作用:进入用户注册页面
+
+4.用户邮件主页面
+url:/userMail
```
Comment (calidion) [inline gj/2017-12-5/api.txt]: userMail
=>
user/mail 或者 mail

File: gj/2017-12-5/api.txt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+
+
+WEB API
+
+GET /users
+1. GET /uers  
+读取用户列表
+
+2. GET /users:id
```
Comment (calidion) [inline gj/2017-12-5/api.txt]: /users:id  
=》
/users/:id

File: gj/2017-12-5/api.txt [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+1. GET /uers  
+读取用户列表
+
+2. GET /users:id  
+读取id对应用户用户的信息
+
+
+POST /users
+1.用户登陆
+url /users
```
Comment (calidion) [inline gj/2017-12-5/api.txt]: url
=>
url:

File: gj/2017-12-5/api.txt [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+mentod:post
+data:action=register
+作用：完成注册
+
+
+GET /mails
+1. GET /mails 
+读取所有邮件列表
+
+2. GET /mails :username
```
Comment (calidion) [inline gj/2017-12-5/api.txt]: /mails :username
=>
/mails/:username

**Comment by ShiJinSi** [comment]
修改

**Comment by ShiJinSi** [comment]
修改2

File: gj/2017-12-5/api.txt [hunk 5]
```diff
@@ -12,19 +12,19 @@ url:/user/register
 作用:进入用户注册页面
 
 4.用户邮件主页面
-url:/userMail
+url:/userhmoe
```
Comment (calidion) [inline gj/2017-12-5/api.txt]: /userhmoe
=>
/user/home

**Comment by ShiJinSi** [comment]
修3

**Pull request merged**